app/src/main/java/com/duckduckgo/app/global/db/AppDatabase.kt

```diff
@@ -60,5 +62,11 @@ abstract class AppDatabase : RoomDatabase() {
                 database.execSQL("CREATE INDEX `index_tab_selection_tabId` on `tab_selection` (tabId)")
             }
         }
+
+        val MIGRATION_2_TO_3: Migration = object : Migration(2, 3) {
```

Comment:
Doubt it will affect many folk, but should the sites visited that are already recorded in the existing table be copied in to the new table?

Happy to merge either way.

Reply:
That's a good point. The old `network_leaderboard` data isn't valid without the total domain count so the percentages will be all over the place. Adding the domains to the new table doesn't fix that as there's still data missing for all the sites visited that didn't have trackers. I've gone ahead and cleared the old `network_leaderboard` data so we can start gathering fresh valid data.